mirdata/annotations.py

```diff
@@ -302,6 +303,61 @@ def to_matrix(
         matrix[index[:, 0], index[:, 1]] = voicing
         return matrix
 
+    def to_multif0(self, time_hop, time_hop_unit, max_time=None):
+        """Convert note annotation to multiple f0 format.
+
+        Args:
+            time_hop (float): time between time stamps in multif0 annotation
+            time_hop_unit (str): unit for time_hop, and resulting multif0 data.
+                One of TIME_UNITS
+            max_time (float, optional): Maximum time stamp in time_hop units.
+                Defaults to None, in which case the maximum note interval
+                time is used.
+
+        Returns:
+            MultiF0Data: multif0 annotation
+        """
+        intervals = convert_time_units(
+            self.intervals, self.interval_unit, time_hop_unit
+        )
+        note_time_max = np.max(intervals[:, 1])
+        max_time = note_time_max if not max_time else max_time
+        if max_time < note_time_max:
+            raise ValueError(
+                "max_time = {} cannot be smaller than the last note interval = {}".format(
+                    max_time, note_time_max
+                )
+            )
+        times = np.arange(0, max_time + time_hop, time_hop)
+        frequency_list: List[list] = [[] for _ in times]
+        confidence_list: Optional[List[list]] = [[] for _ in times]
```

Comment:
If you're gonna add type information here, might as well add it to the function signature?